tools/storybook/.storybook/storybook.css

```
@@ -1,15 +1,33 @@
 /**
  * Storybook Global Styles
  *
- * This file imports Tailwind plus the tiny shared test-token contract and gives
- * those tokens visible defaults inside Storybook.
+ * This file imports Tailwind, the shared token contract, and mirrors the app's
+ * default/georgian font variables inside Storybook.
  */
 
 /* All imports must come first */
+@import url("https://fonts.googleapis.com/css2?family=Manrope:wght@200..800&display=swap");
```

Stylelint (17.12.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=Manrope:wght@200..800&display=swap")" to be ""https://fonts.googleapis.com/css2?family=Manrope:wght@200..800&display=swap"" (import-notation)

(import-notation)